Documentation/community/roadmap.rst

```diff
@@ -146,14 +147,16 @@ CI Test Improvements
 We have a comprehensive set of tests running in CI, but several contributors are
 currently working on `CI improvements`_ to make these more reliable and easier to
 maintain. This is a good area to get involved if you are interested in learning
-more about Cilium internals and development. 
+more about Cilium internals and development.
+
+Tetragon Security
+~~~~~~~~~~~~~~~~~
+
+Tetragon provides security observability through the JSON events and the Tetragon
+CLI for things like process execution, file access, network observability, and
+privileged execution.
 
-Documentation Improvements
-~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-We would like to make this Cilium documentation :ref:`easier to
-navigate<season_of_docs>`, with more example configurations and links to
```

Comment:
Should this PR also remove `Documentation/contributing/development/season_of_docs.rst` and the reference to season of docs from `Documentation/contributing/development/index.rst` ?

Reply:
I was going to remove them in a separate PR once the program is done in December. I'm guessing the graduation application will be in before then, but I'm not sure if it makes sense to have this in here for that

Reply:
That works for me, just wanted to make sure we weren't forgetting to clean up some links. :ok_hand: